README.md

```diff
@@ -1,3 +1,56 @@
-# Locale Switcher
+<p align="center">
+  <img src="./icons/128.png" alt="logo" />
```

Comment:
I think we can remove this old logo from the README

Reply:
Its the new one :)